robin_stringer/test/test.js

```diff
@@ -0,0 +1,20 @@
+var chai = require('chai');
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+var expect = chai.expect;
+var fs = require('fs');
+require(__dirname + '/../app');
+
+describe('the express server', function() {
+  it('should respond to POST requests and return values', function(done) {
+    chai.request('localhost:3000')
+      .post('/note_one')
+      .send('{noteBody: \'hello world\'}')
```

Comment:
It doesn't look like your app.js server is responding to this route (/noteBody). Is this test working?